### ISSUE TITLE
Empty <ul>s in List of Pages block

### ISSUE THREAD

**Issue by erinchambers** [omeka/omeka-s#2518]
Not sure if this has been flagged already—currently, the [List of Pages block](https://github.com/omeka/omeka-s/blob/develop/application/view/common/block-layout/list-of-pages.phtml) is printing an empty `ul` after each page/`li`. Accessibility scans are picking it up as a violation due to invalid HTML ("`<ul>` and `<ol>` must only direction contain `<li>`, ` <script>` or `<template>` elements").

HTML Output:
```html
<div class="block block-listOfPages">
    <div>
        <ul>
            <li><a href="/omeka-s/s/theme-dev/page/welcome">Welcome</a></li>
            <ul></ul>
            <li><a href="/omeka-s/s/theme-dev/page/test-pages">Test Pages</a></li>
            <ul></ul>
            <li><a href="/omeka-s/s/theme-dev/page/blocks-1">Blocks 1</a></li>
            <ul></ul>
            <li><a href="/omeka-s/s/theme-dev/page/blocks-2">Blocks 2</a></li>
            <ul></ul>
        </ul>
    </div>
</div>
```
